SingularityService/src/main/java/com/hubspot/singularity/guice/BootstrapAwareModule.java

```diff
@@ -0,0 +1,29 @@
+package com.hubspot.singularity.guice;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.base.Preconditions.checkState;
+
+import com.google.inject.Binder;
+import com.google.inject.Module;
+
+import io.dropwizard.setup.Bootstrap;
+
+public abstract class BootstrapAwareModule implements Module {
+  private Bootstrap<?> bootstrap = null;
```

Comment:
`ConfigurationAwareModule` (which this is based off of) had this variable marked as `volatile` -- not sure if necessary or not